low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python3
+#!/usr/bin/python3.6
```

Comment:
How about using following then, Rocky linux doesnt have soft link for python3 ? Its similar to code we have today. 
**#!/usr/bin/python3**

>cortx-monitor$ ls -l /usr/bin/python*
lrwxrwxrwx. 1 root root     7 Feb 11  2020 /usr/bin/python -> python2
lrwxrwxrwx. 1 root root     9 Feb 11  2020 /usr/bin/python2 -> python2.7
-rwxr-xr-x. 1 root root  7216 Aug  6  2019 /usr/bin/python2.7
-rwxr-xr-x  1 root root   304 May  3  2019 /usr/bin/python2.7-futurize
-rwxr-xr-x  1 root root   308 May  3  2019 /usr/bin/python2.7-pasteurize
lrwxrwxrwx  1 root root     9 Jul 17  2020 /usr/bin/python3 -> python3.6
-rwxr-xr-x  2 root root 11336 Apr  2  2020 /usr/bin/python3.6
lrwxrwxrwx  1 root root    17 Jul 17  2020 /usr/bin/python3.6-config -> python3.6m-config
-rwxr-xr-x  2 root root 11336 Apr  2  2020 /usr/bin/python3.6m
-rwxr-xr-x  1 root root   173 Apr  2  2020 /usr/bin/python3.6m-config
-rwxr-xr-x  1 root root  3403 Apr  2  2020 /usr/bin/python3.6m-x86_64-config
lrwxrwxrwx  1 root root    16 Jul 17  2020 /usr/bin/python3-config -> python3.6-config
-rwxr-xr-x  1 root root  2558 Apr  2  2019 /usr/bin/python-argcomplete-check-easy-install-script
-rwxr-xr-x  1 root root   234 Nov 30  2016 /usr/bin/python-argcomplete-tcsh
cortx-monitor$

**Linking to python minor version may need us to update code again if future OS have other minor version, so we cant go with 3.6**

Reply:
Yes, Completely Agree. Kept all the shebang in sspl as `/usr/bin/python3`

Reply:
The alternative of `python` and `python3` is set to `/usr/bin/python3.6` that's why using `/usr/bin/python3` as shebang is possible and feasible. 
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#python_dynamic-programming-languages-web-servers-database-servers